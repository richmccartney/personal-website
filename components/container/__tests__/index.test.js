import React from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'
import { matchers } from 'jest-emotion'

import Container from '../index'

expect.extend(matchers)

describe('Container:', () => {
  const wrapper = mount(<Container>Hello world!</Container>)

  test('renders snapshot as expected', () => {
    const component = renderer.create(<Container />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
    expect(tree).toHaveStyleRule('grid-column-start', '2')
    expect(tree).toHaveStyleRule('grid-column-start', '2')
  })

  it('renders as <div> as expected', () => {
    const container = wrapper.find('div')
    expect(container.is('div')).toBe(true)
  })
})
