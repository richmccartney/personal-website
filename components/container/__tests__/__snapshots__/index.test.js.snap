// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container: renders snapshot as expected 1`] = `
<div
  className="css-1z08k44"
/>
`;
